src/home/buttons/camera.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Button, Grid } from '@material-ui/core'
+import { useNavigation } from 'home/buttons/hooks/use-navigation'
+
+export const Camera = ({ children }) => <Grid item>
```

Comment:
Why pass in children here? We are not creating a wrapper or a reusable component. So there are two options here:
* Create a common, reusable component (called something like `NavigationButton` or whatever) and pass in props to that component
* Hard code the labels within each button component.

I'll let you figure out which one is the better option :smile:

Reply:
done